cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -3089,30 +3091,47 @@ def test_api_v2_tasks_id_export_no_auth(self):
         self._run_api_v2_tasks_id_export_import(None)
 
     def test_can_remove_export_cache_automatically_after_successful_export(self):
+        from cvat.apps.engine.cron import cron_export_cache_cleanup, clear_export_cache
         self._create_tasks()
         task_id = self.tasks[0]["id"]
         user = self.admin
 
-        with mock.patch('cvat.apps.dataset_manager.views.TASK_CACHE_TTL', new=timedelta(hours=10)):
+        TASK_CACHE_TTL = timedelta(seconds=5)
+        with (
+            mock.patch('cvat.apps.dataset_manager.views.TASK_CACHE_TTL', new=TASK_CACHE_TTL),
+            mock.patch('cvat.apps.dataset_manager.views.TTL_CONSTS', new={'task': TASK_CACHE_TTL}),
+            mock.patch(
+                "cvat.apps.engine.cron.clear_export_cache",
+                side_effect=clear_export_cache,
+            ) as mock_clear_export_cache,
+            mock.patch(
+                "cvat.apps.engine.cron.get_current_job",
+            ) as mock_rq_get_current_job,
+        ):
+            mock_rq_job = mock.MagicMock(timeout=100)
+            mock_rq_get_current_job.return_value = mock_rq_job
+            cron_export_cache_cleanup()
```

Comment:
With 5 seconds as export cache TTL the test will be working unreliable on slow machines. It's better to either patch the expiration check or use some significant number as TTL, e.g. 1 hour.